lux/lib/lux/lenses/discord/channels/read_channel.ex

```diff
@@ -0,0 +1,70 @@
+defmodule Lux.Lenses.Discord.Channels.ReadChannel do
```

Comment:
Having too many stacked PRs might be a problem later. These were all independent so it would have been better to make them all fork `Spectral-Finance:main`